.annotation_safe_list.yml

```diff
@@ -0,0 +1,41 @@
+# This is a Code Annotations automatically-generated Django model safelist file.
```

Comment:
why do we need this?

Reply:
It's for quality checks, Those fail if we don't have this file defined

Reply:
Then we should remove those quality checks as we've done in the other repos